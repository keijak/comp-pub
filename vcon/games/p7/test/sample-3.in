5 7 9
BRRBR
3 1
5 2
4 2
2 1
5 4
5 1
3 2
