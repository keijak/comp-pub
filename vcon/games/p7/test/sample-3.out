Second
First
First
Second
First
